service_container/service_decoration.rst

```diff
@@ -131,7 +131,9 @@ but keeps a reference of the old one as ``.inner``:
     The ``#[AsDecorator]`` attribute was introduced in Symfony 6.1.
 
 The ``decorates`` option tells the container that the ``App\DecoratingMailer``
-service replaces the ``App\Mailer`` service. If you're using the
+service replaces the ``App\Mailer`` service. 
+:ref:`Service tags<How to Work with Service Tags> are moved as well. 
```

Comment:
If I understand well symfony codebase, not all tags are kept

https://github.com/symfony/symfony/blob/7de25db56093a53b0b2d85f96d716ff6e9bf7147/src/Symfony/Component/DependencyInjection/Compiler/DecoratorServicePass.php#L49-L52

https://github.com/symfony/symfony/blob/7de25db56093a53b0b2d85f96d716ff6e9bf7147/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L739-L748